plugins/modules/pnp_intent.py

```diff
@@ -818,16 +852,18 @@ class instance for further use.
                     params=claim_params,
                 )
 
-                self.log(str(claim_response))
+                self.log("Single claiming response from the API 'claim a device to a site' is {0}".format(str(dev_add_response)), "DEBUG")
```

Comment:
self.log("Response from API 'claim a device to a site' for a single claiming: {0}".format(str(dev_add_response)), "DEBUG")

Reply:
Resolved